src/Service.Tests/GraphQLBuilder/Sql/StoredProcedureBuilderTests.cs

```diff
@@ -53,6 +53,7 @@ public class StoredProcedureBuilderTests
         [DataRow(typeof(DateTime), DATETIME_TYPE, "12/31/2030 12:00:00 AM", false, DisplayName = "DateTime")]
         [DataRow(typeof(DateTime), DATETIME_TYPE, "12/31/2030 12000 AM", true, DisplayName = "DateTime")]
         [DataRow(typeof(DateTimeOffset), DATETIME_TYPE, "11/19/2012 10:57:11 AM -08:00", false, DisplayName = "DateTimeOffset")]
+        [DataRow(typeof(TimeSpan), TIMESPAN_TYPE, "10:57:11.0000", false, DisplayName = "TimeSpan")]
```

Comment:
HotChocolate supports string literals only in ISO-8601 duration whereas SQL Server supports the following two additional formats (apart from the ones in ISO-8601)
`hhAM[PM]` and `hh AM[PM]`

[HotChocolate .NET Scalar Docs](https://chillicream.com/docs/hotchocolate/v13/defining-a-schema/scalars#datetime-type)
[Sql Server's time datatype Docs](https://learn.microsoft.com/en-us/sql/t-sql/data-types/time-transact-sql?view=sql-server-ver16)

Could be worth validating the other string formats and adding test cases

Reply:
added a test for both.